### DISCUSSION TITLE
How do you edit the theme? (question)

### DISCUSSION THREAD

**Discussion by ghost** [BEEmod/BEE2-items#4192] [Tech Support / Q&A] [answered]
Hi, I've got BEEmod 4.36.1 and I cant find a way to edit the theme of the chamber upon export. Any Help Appreciated!

**Comment by vrad-exe** [comment, marked as answer]
What do you mean? There's a style selector in the export options window.

**Comment by ghost** [comment]
@Luke18033 Thanks, I'm surprised I didn't look there
